soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -484,121 +484,121 @@ void InitTrickNames() {
-     Text{"Heart Container", "Réceptacle de coeur", "Contenedor de corazón"},
-     Text{"Piece of Heart", "Quart de coeur", "Pieza de corazón"}};
+     Text{"Life", "Vie", "vida"},
+     Text{"HP", "VP", "VP"}};
   trickNameTable[GI_RUPEE_GREEN] = {
-     Text{"Green Rupy", "Rupee vert", "Rubia verde"},
-     Text{"One Rupee", "Un rubis", "Guaraní hyliano"},
-     Text{"Rupee (1)", "Rubis (1)", "Peso hyliano"}};
+     Text{"False Greg", "Faux Greg", "Falso Greg"},
+     Text{"One Ruby", "Un rubis", "Un rubí"},
+     Text{"Rupoor (1)", "Roupir (1)", "Rupobre (1)"}};
   trickNameTable[GI_RUPEE_BLUE] = {
-     Text{"Blue Rupy", "Rupee bleu", "Rubia azul"},
-     Text{"Five Rupees", "Cinq rubis", "Bolívar hyliano"},
-     Text{"Rupee (5)", "Rubis (5)", "Peso hyliano"}};
+     Text{"Blupee", "Bleubi", "Azupia"},
+     Text{"Five Rubies", "Cinq Rubys", "Cinco rubíes"},
+     Text{"Rupoor (5)", "Roupir (5)", "Rupobre (5)"}};
   trickNameTable[GI_RUPEE_RED] = {
-     Text{"Red Rupy", "Rupee rouge", "Rubia roja"},
-     Text{"Twenty Rupees", "Vingt rubis", "Colon hyliano"},
-     Text{"Rupee (20)", "Rubis (20)", "Peso hyliano"}};
+     Text{"Big 20", "Grand 20", "Los 20 grandes"},
+     Text{"Twenty Rubies", "vingt rubis", "Veinte rubíes"},
+     Text{"Rupoor (20)", "Roupir (20)", "Rupobre (20)"}};
   trickNameTable[GI_RUPEE_PURPLE] = {
-     Text{"Purple Rupy", "Rupee pourpre", "Rubia morada"},
-     Text{"Fifty Rupees", "Cinquante rubis", "Balboa hyliano"},
-     Text{"Rupee (50)", "Rubis (50)", "Peso hyliano"}};
+     Text{"Purpee", "pourbi", "morupiua"},
+     Text{"Fifty Rubies", "cinquante rubis", "Cincuenta rubíes"},
+     Text{"Rupoor (50)", "Roupir (50)", "Rupobre (50)"}};
   trickNameTable[GI_RUPEE_GOLD] = {
-     Text{"Huge Rupy", "Énorme Rupee", "Rubia gigante"},
-     Text{"Two Hundred Rupees", "Deux cent rubis", "Euro hyliano"},
-     Text{"Rupee (200)", "Rubis (200)", "Dólar hyliano"}};
+     Text{"Hugo", "Or Rubi", "Oro Rubi"},
+     Text{"Two Hundred Rubies", "deux cents rubis", "Doscientos rubíes"},
+     Text{"Diamond", "Diamant", "Diamante"}};
   trickNameTable[GI_HEART_PIECE] = {
-     Text{"Piece of Health", "Quart d'énergie", "Pieza de amor"},
-     Text{"Recovery Heart", "Coeur d'énergie", "Corazón"},
-     Text{"Heart Container", "Réceptacle de coeur", "Contenedor de corazón"}};
+     Text{"Pizza Heart", "Fromage de cœur", "Pieza de Chorizo"},
+     Text{"Little Bit Of Love", "Un peu d'amour", "Un poco de amor"},
+     Text{"Rare Peach Stone", "Pierre de pêche rare", "Pierre de pêche rare"}};
   trickNameTable[GI_HEART_CONTAINER_2] = {
-     Text{"Health Container", "Réceptacle d'énergie", "Contenedor de amor"},
-     Text{"Recovery Heart", "Quart de coeur", "Corazón"},
-     Text{"Piece of Heart", "Coeur d'énergie", "Pieza de corazón"}};
+     Text{"Crystal Heart", "Cœur de cristal", "Corazón de cristal"},
+     Text{"Life Heart", "Cœur de vie", "Vida Corazón"},
+     Text{"Lots of Love", "Beaucoup d'amour", "Mucho amor"}};
```

Comment:
Same here, old ones were better

Reply:
Because it's not obvious, "Diamond" is a Zelda's adventure reference (where it's a special item that maxes your money) and the top row on the rupees are rando meme names.